content/03-type-system.md

```diff
@@ -1,31 +1,31 @@
 
-A lot of type-safety is achieved by checking if two given types of the type groups above are compatible. Meaning, the compiler tries to perform **unification** between them as detailed in [Unification](type-system-unification).
+ !!!-REVIEW ME: What is this sentence even referring to? "Type groups" is also unclear in the sense that this phrase has not been used before and may have special meaning of some sort-!!! A lot of type-safety is achieved by checking if two given types of the type groups above are compatible. Meaning, the compiler tries to perform **unification** between them as detailed in [Unification](type-system-unification).
 
 All types are organized in **modules** and can be addressed through **paths**. [Modules and Paths](type-system-modules-and-paths) will give a detailed explanation of the related mechanics.
 
 <!--label:type-system-typedef-->
 ### Typedef
 
-We briefly looked at typedefs while talking about [anonymous structures](types-anonymous-structure) and saw how we could shorten a complex [structure type](types-anonymous-structure) by giving it a name. This is precisely what typedefs are good for. Giving names to structure types might even be considered their primary use. In fact, it is so common that the distinction appears somewhat blurry and many Haxe users consider typedefs to actually **be** the structure.
+We briefly looked at typedefs while talking about [anonymous structures](types-anonymous-structure) and saw how we could shorten a complex [structure type](types-anonymous-structure) by giving it a name. This is precisely why typedefs are useful. Giving names to structure types might even be considered their primary function, and is so common that the distinction between the two appears somewhat blurry. Many Haxe users consider typedefs to actually **be** the structure.
 
 A typedef can give a name to any other type:
 
 ```haxe
 typedef IA = Array<Int>;
 ```
-This enables us to use `IA` in places where we would normally use `Array<Int>`. While this saves only a few keystrokes in this particular case, it can make a much bigger difference for more complex, compound types. Again, this is why typedef and structures seem so connected:
+This enables us to use `IA` in places where we would normally use `Array<Int>`. While this saves only a few keystrokes in this particular case, it can make a larger difference for more complex, compound types. Again, this is why typedef and structures seem so connected:
 
 ```haxe
 typedef User = {
   var age : Int;
   var name : String;
 }
 ```
-A typedef is not a textual replacement but actually a real type. It can even have [type parameters](type-system-type-parameters) as the `Iterable` type from the Haxe Standard Library demonstrates:
+A typedef is not a textual replacement, but actually a real type. They can even have [type parameters](type-system-type-parameters) as the `Iterable` type from the Haxe Standard Library demonstrates:
```

Comment:
Going from singular "A typedef" to plural "they" sounds strange to me. Maybe this paragraph should start plural?

Reply:
Your change sounds appropriate. I should have noticed beforehand really with how much this paragraph annoyed me :)